### ISSUE TITLE
UX: Make recommendation for "Where can I find these numbers?" tab in payments

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7813]

**Comment by wavelaurenrussell** [comment]
Replace [Where can I find these numbers] with link component; put the expanded content in a large panel.

**Comment by brea11y** [comment]
Ticket has been reviewed and is still valid.

**Comment by timwright12** [comment]
@JonathanPostMO can you review this one to see if we'd like to get it into the app?

**Comment by JonathanPostMO** [comment]
@timwright12 The screenshot is broken, "Non-Image content-type returned". This is just the image of the check on the Direct Deposit page though, the "where to find your banking info" essentially?

**Comment by timwright12** [comment]
@JonathanPostMO I believe so, yeah. But if the broken image makes this not actionable we can close too

**Comment by JonathanPostMO** [comment]
@timwright12 We might to push the Benefits team to do a research spike on. I'm not sure using a single image of a paper check is the most accessible way to explain this, especially when it might be easier for some users to copy-paste their banking info from their banking app? It's got some mutlimodal spice, there's a lot of ways for users to find their banking info. 

Maybe the overall UX needs modernization? Want to punt to @BeccaTupaj for input too.

**Comment by timwright12** [comment]
@JonathanPostMO I updated the broken image

Possibly, but for the scope this what this ticket is referencing, would you agree that we should replace the accordion with a link that displays the image in a panel? If so, we can move this forward and then break off another work stream to address the actual experience

**Comment by BeccaTupaj** [comment]
I agree with @timwright12 's recommendation here. Lets keep the solution here focused on this smaller issue and we can consider larger UX improvements in another epic.
